src/content/docs/en/guides/media/mux.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: Mux & Astro
+description: Add high performance video to your Astro project using Mux
+sidebar:
+  label: Mux
+type: media
+service: Mux
+stub: true
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+[Mux](https://mux.com) is a hosted media service that provides video streaming infrastructure and performance analytics for businesses of all scale.
+
+When you use Mux to store and host your video content, you'll have access to Astro-native video components for [Mux Player](#mux-player), a drop in component for adding Mux videos in your Astro project, and [Mux Uploader](#mux-uploader) for uploading videos to Mux from your website. These components integrate seamlessly with [Mux Data](https://www.mux.com/docs/guides/data) to track your video engagement and performance.
+
+You can also interact with your content through the [Mux Node SDK](#mux-node-sdk).
+
+:::tip
+Learn more about features such as embedding, storing, streaming, and customizing video at [Mux's dedicated page for video in Astro](https://www.mux.com/video-for/astro)!
```

Comment:
Add offer code here?

Reply:
The only other place I considered it was in the "Prerequisites" section where we already direct people to sign up for a Mux account:

> - A Mux account. If you don’t have an account, you can [sign up with Mux](https://dashboard.mux.com/login) for a free account.

Strong preference for one vs. the other? It maybe feels more natural where we send people to log in or sign up than the tip, but I can understand why Mux might like it in a tip. Thoughts?

Reply:
Will put it in the prerequisites section!